### PR TITLE
add dino

### PR DESCRIPTION
**Pull request by xperzy**
Add dino, now the weight decay scheduler is not supported, since the paddle framework does not support wd settings for optimizer. 

The model training is runnable but the training results is not provided now

### CODE REVIEW

**Comment by xperzy** [comment]
The wd scheduler is not supported and corrected set.
